webapp/WEB-INF/templates/admin/plugins/forms/composite/view_group.html

```diff
@@ -1,5 +1,40 @@
-<#assign fieldsetClass=''>
-<#if id_display??><#assign fieldsetClass='display_field_${id_display}'></#if>
-<@fieldSet class='fieldset-group ${fieldsetClass}' legend=group.title fieldsetId='group-${group.id}' fieldsetParams=' tabindex=0'>
-	<#list groupContent as childrenHtml>${childrenHtml}</#list>
-</@fieldSet>
+<#assign indexGroup=-1>
+<#assign indexIteration=0>
+<#assign nbIterationMin=group.iterationMin>
+<@cSection type='div' class='step-group'>
+<#list groupContent as childrenHtml>
+	<#assign indexIteration = ( childrenHtml?index / nbBaseChildren )?int />
+	<#assign nbTotalIteration = ( groupContent?size / nbBaseChildren )?int />
+	<#if indexGroup != indexIteration>
+		<#if childrenHtml?is_last><a name="last_${id_group}"></a></#if>
+		<#assign cClass><#if childrenHtml?index &gt;= 1>border-top col-12 pt-2</#if><#if id_display??> display_field_${id_display} hidden</#if></#assign>
+		<#if cClass !=''><div class="${cClass}"></#if>
+		<@cHeader class'col-12 d-flex justify-space-between'>
+			<@cTitle class='flex-fill'>${group.title} <#if indexIteration gt 0>(${indexIteration+1})</#if> </@cTitle>
+			<#if nbTotalIteration gt nbIterationMin>
+				<@cBtn class='link-action' label='' params=' formnovalidate name="action_removeIteration" value="${group.id}_${indexIteration}"'>
+					<@cIcon label='' class='trash' /> <small class="d-inline">${group.iterationRemoveLabel!''}</small>
+				</@cBtn>
+			</#if>
+			<#if isIterable && indexIteration = nbTotalIteration - 1>
+				<@cBtn class='action btn-sm ml-2' label='' params='formnovalidate value="${id_group}" name="action_addIteration" '>
+					<@cIcon class='plus-circle' label='' /> ${group.iterationAddLabel!}
+				</@cBtn>
+			</#if>
+		</@cHeader>
+		<#if cClass !=''></div></#if>
+		<div class="d-flex flex-wrap">
+	</#if>
+	<#if group.description!='' && indexGroup=-1>
+		<@cFormRow>
+			<@cCol>
+				<@cAccordion title='Aide sur ${group.title}' id='helpInfo${indexIteration}' class='outline my-5 w-100' >${group.description}</@cAccordion>
```

Comment:
Please use the i18n files for the labels, if you would.

Reply:
Done